packages/colony-starter-basic/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@colony/colony-starter-basic",
+  "version": "1.0.0",
+  "description": "A simple starter project for building with Colony",
+  "license": "MIT",
+  "engines": {
+    "node": "9.5"
+  },
+  "scripts": {
+    "initialize": "sh scripts/initialize_project.sh",
```

Comment:
Do these shell scripts run okay on Windows? I'm only asking because I have literally no idea how Windows copes with bash stuff these days

Reply:
I'm not sure. We should definitely get someone running Windows to double check. Do you know of anyone that could help us out here?